documentation/specs/proposed/General_perf_onepager.md

```diff
@@ -0,0 +1,42 @@
+# General performance improvements
+MSBuild is the main tool used for building .NET projects. It is used by Visual Studio, the .NET CLI, and other build tools. The performance of MSBuild is critical to the productivity of developers. This document outlines our goals to improve overall performance of MSBuild.
+
+## Goals and Motivation
+
+We are aiming for:
+ - Searching opportunities to improve the build times.
```

Comment:
I would pick build **and** evaluation time. Both are heavily important for Visual Studio.

Reply:
I considered evaluation as part of the build internally, but I will express it explicitly.

Reply:
We do in-procs evaluations without builds.